.gitignore

```diff
@@ -1,3 +1,7 @@
+app.db
+migration/
+migrations/
```

Comment:
Is it normal to have migration and migrations ? Is there a difference between them ?

Reply:
Does not cost much to have a doublon there.
Was not sure of the type and would prefer not to have a migration and migrations directory, with one pushable and the other not.